src/main/java/org/alfresco/util/ConfigFileFinder.java

```diff
@@ -155,6 +155,11 @@ private void readFromDisk(String path, Log log, AtomicBoolean successReadingConf
             Arrays.sort(files, (file1, file2) -> file1.getName().compareTo(file2.getName()));
             for (File file : files)
             {
+                if (file.isDirectory())
+                {
+                    log.debug("Skipping directory " + file.getName() + " in " + path);
+                    continue;
```

Comment:
Can we please have an else clause rather than a continue. There is nothing else in the loop, so is better style.

Reply:
Done